docs/src/examples.md

```diff
@@ -750,61 +750,106 @@ edge states also in high-dimensional and chaotic systems where a simple computat
     du[2] = x
     du[3] = C - u^2 + D*v + k*(u-x)
     du[4] = u
-    return 
+    return
 end
-ds = DeterministicIteratedMap(F!, zeros(4)) 
+ds = DeterministicIteratedMap(F!, zeros(4))
 ```
 
-Next we define a region in the phase space that should not contain attractors. 
-Using this region we also define a `sampler` and a membership function `isinside`: 
+Next we define a region in the phase space that should not contain attractors.
+Using this region we also define a `sampler` and a membership function `isinside`:
 
 ```@example MAIN
-R_min = [-4; -4.; -4.; -4.] 
+R_min = [-4; -4.; -4.; -4.]
 R_max = [4.; 4.; 4.; 4.]
 sampler, isinside = statespace_sampler(HRectangle(R_min,R_max))
 ```
 
-And we are ready! We can now call the function `stagger_and_step` with an initial 
-condition `x0`: 
+And we are ready! We can now call the function `stagger_and_step` with an initial
+condition `x0`:
 
 ```@example MAIN
 x0 = sampler()
-v = stagger_and_step(ds, x0, 10000, isinside; stagger_mode = :adaptive, δ = 1e-4, Tm = 10, max_steps = Int(1e5), δ₀ = 2.) 
+v = stagger_and_step(ds, x0, 10000, isinside; stagger_mode = :adaptive, δ = 1e-4, Tm = 10, max_steps = Int(1e5), δ₀ = 2.)
 ```
-The `stagger_mode` keyword select the type of search in the 
-phase space to stick close to the saddle at each step. 
+The `stagger_mode` keyword select the type of search in the
+phase space to stick close to the saddle at each step.
 The mode `:adaptive` adapts the radius of the stochastic
-search as a function of the success of the search process. 
+search as a function of the success of the search process.
 
-Finally we can represent a projection of the chaotic saddle found in this 
-example: 
+Finally we can represent a projection of the chaotic saddle found in this
+example:
 
 ```@example MAIN
 fig = Figure()
 ax = Axis(fig[1,1], xlabel="x", ylabel="y")
 scatter!(ax, v[:,1], v[:,3]; markersize = 3)
 fig
 ```
+
+## Matching limit cycles and fixed points in a system with heterogeneous state space
+
+This example discusses the situation of a dynamical system that during a global continuation
+transitions from a fixed point to a limit cycle and then to another fixed point that is on
+the other end of the limit cycle. In the context of this scenario, we want to NOT
+match the fixed points with the limit cycle during the continuation.
+Furthermore, this particular dynamical system has a heterogeneous state space:
+the different dynamic variables have wildly different units, and there is no sensible transformation
+that would bring all variables to the same units.
+
+We will showcase how one can achieve match attractors in this system simply by defining a
+special distance function that is given to [`MatchBySSSetDistance`](@ref). This is:
+
+```@example MAIN
+function centroid_and_length(A, B)
+    # first check we are comparing a fixed point and limit cycle. We do this by
+    # checking if there are different lengths and one of the two is 1 (fixed point)
```

Comment:
"We do this by checking if the lengths of sets A and B are different and if one of the sets has length 1 (fixed point)"